src/AggregateNugetPackages/FxCop/NuGet/FxCopAnalyzers.NuGet.proj

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The FxCop nuget files just moved to a sub-folder - contents are same as before here.